tools/docker/openems-backend/.atom-build.yml

```diff
@@ -0,0 +1 @@
+cmd: "docker build -t stromdao/openems-backend .;docker push stromdao/openems-backend"
```

Comment:
This seems to be stromdao specific. Can we remove this file and instead add the command as a hint to the readme - like:

## Build
`docker build -t stromdao/openems-backend .; docker push stromdao/openems-backend`